tests/ts_simple_select.erl

```diff
@@ -27,14 +27,18 @@
 -export([confirm/0]).
 
 confirm() ->
-    TestType = normal,
-    DDL = ts_util:get_ddl(),
-    Data = ts_util:get_valid_select_data(),
-    Qry = ts_util:get_valid_qry(),
+    Table = ts_data:get_default_bucket(),
+    DDL = ts_data:get_ddl(),
+    Data = ts_data:get_valid_select_data(),
+    Qry = ts_data:get_valid_qry(),
     Expected =
-        {ok, {ts_util:get_cols(small),
-         ts_util:exclusive_result_from_data(Data, 2, 9)}},
-    Got = ts_util:ts_query(
-            ts_util:cluster_and_connect(single), TestType, DDL, Data, Qry),
+        {ok, {ts_data:get_cols(small),
+         ts_data:exclusive_result_from_data(Data, 2, 9)}},
+
+    Cluster = ts_setup:start_cluster(1),
+    ts_setup:create_bucket_type(Cluster, DDL, Table),
+    ts_setup:activate_bucket_type(Cluster, Table),
+    ts_ops:put(Cluster, Table, Data),
+    Got = ts_ops:query(Cluster, Qry),
     ?assertEqual(Expected, Got),
```

Comment:
@andytill Code maintenance is a huge issue. If each test has a static schema and data, then each test individually needs to be updated when a sweeping change happens, rather than updating this in one library.  Sadly we've had several big changes over the life of TS.  Many of the functions in the utility library are simply setting up a cluster into an interesting test state, but the details are less important.  Much like building a cluster.  I would agree that the interesting features of what are being tested should be in the test, however.